daemon/restapi/api_limits_test.go

```diff
@@ -13,8 +13,11 @@ import (
 	"github.com/stretchr/testify/assert"
 	timeRate "golang.org/x/time/rate"
 
+	"github.com/cilium/hive/hivetest"
+
```

Comment:
nit: unnecessary empty line

Reply:
The canonical `goimports -w -local github.com/cilium/cilium` does not change this. If I manually clean up the space it's also happy, but I'm not super excited to discuss formatting if it's not tool-enforced.

Don't get me wrong, happy to clean this one up, but in general my standpoint is mostly "if the linters don't complain, it's good enough"